src/test/rgw/rgw_multi/tests_ps.py

```diff
@@ -2175,1897 +1104,645 @@ def test_ps_s3_persistent_notification_pushback():
     http_server.close()
 
 
-def test_ps_s3_notification_push_kafka():
```

Comment:
"push" tests that don't end with "on_master()" must stay in this file

Reply:
Okay.